packages/astro/src/@types/astro.ts

```diff
@@ -1555,10 +1555,19 @@ export interface AstroUserConfig {
  */
 export type InjectedScriptStage = 'before-hydration' | 'head-inline' | 'page' | 'page-ssr';
 
+/**
+ * IDs for different priorities of injected routes and redirects:
```

Comment:
I am not really sure redirects should be configured. I think redirects should have the highest priority above all routes.

Reply:
It is documented that they have the lowest priority of all the routes, so I didn't change that default. I personally think both of them should have the same priority as the project routes by default, but that would break way too many things.

I wasn't going to make them configurable, but it was pointed out on Discord that there are similar problems for them as there are for injected routes (albeit inversed since they go below the project, not above), so I fixed them together here.